src/main/java/com/algorand/algosdk/v2/client/common/AlgodClient.java

```diff
@@ -27,108 +27,108 @@ public AlgodClient(String host, int port, String token) {
+     * transactions.
+     * /v2/accounts/{address}/transactions/pending
      */
     public GetPendingTransactionsByAddress GetPendingTransactionsByAddress(Address address) {
         return new GetPendingTransactionsByAddress((Client) this, address);
     }
 
     /**
-     * /v2/blocks/{round} 
+     * /v2/blocks/{round}
      */
     public GetBlock GetBlock(Long round) {
         return new GetBlock((Client) this, round);
     }
 
     /**
-     * /v2/ledger/supply 
+     * /v2/ledger/supply
      */
     public GetSupply GetSupply() {
         return new GetSupply((Client) this);
     }
 
     /**
-     * /v2/status 
+     * /v2/status
      */
     public GetStatus GetStatus() {
         return new GetStatus((Client) this);
     }
 
     /**
-     * Waits for a block to appear after round {round} and returns the node's status at 
-     * the time. 
-     * /v2/status/wait-for-block-after/{round} 
+     * Waits for a block to appear after round {round} and returns the node's status at
+     * the time.
+     * /v2/status/wait-for-block-after/{round}
      */
     public WaitForBlock WaitForBlock(Long round) {
         return new WaitForBlock((Client) this, round);
     }
 
     /**
-     * /v2/transactions 
+     * /v2/transactions
      */
     public RawTransaction RawTransaction() {
         return new RawTransaction((Client) this);
     }
 
     /**
-     * /v2/transactions/params 
+     * /v2/transactions/params
      */
     public TransactionParams TransactionParams() {
         return new TransactionParams((Client) this);
     }
 
     /**
-     * Get the list of pending transactions, sorted by priority, in decreasing order, 
-     * truncated at the end at MAX. If MAX = 0, returns all pending transactions. 
-     * /v2/transactions/pending 
+     * Get the list of pending transactions, sorted by priority, in decreasing order,
+     * truncated at the end at MAX. If MAX = 0, returns all pending transactions.
+     * /v2/transactions/pending
      */
     public GetPendingTransactions GetPendingTransactions() {
         return new GetPendingTransactions((Client) this);
     }
 
     /**
-     * Given a transaction id of a recently submitted transaction, it returns 
-     * information about it. There are several cases when this might succeed: 
-     * - transaction committed (committed round > 0) - transaction still in the pool 
-     * (committed round = 0, pool error = "") - transaction removed from pool due to 
-     * error (committed round = 0, pool error != "") 
-     * Or the transaction may have happened sufficiently long ago that the node no 
-     * longer remembers it, and this will return an error. 
-     * /v2/transactions/pending/{txid} 
+     * Given a transaction id of a recently submitted transaction, it returns
+     * information about it. There are several cases when this might succeed:
+     * - transaction committed (committed round > 0) - transaction still in the pool
```

Comment:
missing newlines here also

Reply:
Is any new line dropped in this PR? Or this is an existing issue?
If this is not caused in this PR, I rather fix it in another PR.